rankboost.py

```diff
@@ -436,6 +436,42 @@ def prepare_dataframe(mhc, predfile, expfile, pepfile, efilt_threshold, boost):
     _log.info('Finished creating the data frame for rank boosting..')
     return all_data, stats, boost
 
+def collapse_iars (stats):
+    """
+    Collapse duplicated IARs.
+    :param stats:
+    :return:
+    """
+    for pep_name_o in stats.index:
```

Comment:
I also realize that you can create a new column in the dataframe called extended_group = group if uncollapsed else ','.join(items). Then you change very very little in terms of other code except the final print stage where you print collapsed_groups insread of groups. makes sense?